jjb/performance-testing/performance-test.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+- project:
+    name: performance-testing
+    project-name: performance-testing
```

Comment:
There is currently a "performance-test" repo in edgexfoundry-holding, but it does not have a Jenkinsfile.

Reply:
Sorry, I didn't point out the Jenkinsfile is on reviewing, please see branch [issue-7](https://github.com/edgexfoundry-holding/performance-test/tree/issue-7).